charts/prerequisites/values.yaml

```diff
@@ -27,14 +27,56 @@ elasticsearch:
       cpu: "1000m"
       memory: "1024M"
 
-  # # Request smaller persistent volumes.
+  # Request smaller persistent volumes.
   # volumeClaimTemplate:
   #   accessModes: ["ReadWriteOnce"]
   #   storageClassName: "standard"
   #   resources:
   #     requests:
   #       storage: 100M
 
+opensearch:
+  enabled: false
+
+  # If you're running in production, set this to false, replicas to 3 and uncomment antiAffinity below
+  # Or alternatively if you're running production, bring your own OpenSearch
+  singleNode: true
+  # replicas: 3
+  # antiAffinity: "hard"
+
+  # By default security is enabled for OpenSearch, disable it here.
+  config:
+    opensearch.yml: |
+      plugins:
+        security:
+          disabled: true
+
+  extraEnvs:
+    - name: DISABLE_INSTALL_DEMO_CONFIG
+      value: "true"
+
+  image:
+    tag: "2.11.0"
+
+  # Shrink default JVM heap.
+  opensearchJavaOpts: "-Xmx512M -Xms512M"
+
+  # Allocate smaller chunks of memory per pod.
+  resources:
+    requests:
+      cpu: "100m"
+      memory: "1024M"
+    limits:
```

Comment:
Suggest not copying the `limits` from the ES section, CPU limits are usually not necessary. Increase `cpu` requests to like "1000m".

Reply:
I just wanted to have the same limited resources like Elasticsearch, but I think in this case we can even remove the entire resources config - the OpenSearch Helm chart already includes a working config: https://github.com/opensearch-project/helm-charts/blob/f4adc91e9435b0d697237bf6003c8aef7cf20fe3/charts/opensearch/values.yaml#L146-L151

You shouldn't really use these very limited resource limits for Elasticsearch (or OpenSearch) in production anyway...